templates/RA.html.tmpl

```diff
@@ -423,8 +422,9 @@
 				{{#each data.sections.sec5.risks }}
 				<tr>
 					<td class="lean">{{key}}</td>
-					<td class="lean">{{{requirementsKey}}}</td>
-					<td class="content-wrappable">{{{requirements}}}</td>
+                    {{#each requirements}}
+                      <td class="content-wrappable">{{key}}: {{name}}<br>{{{description}}}</td>
```

Comment:
Please, use <br/>. Even though this is an HTML template, we want it to be compliant with XHTML.